runtime/src/main/java/org/corfudb/runtime/object/VersionLockedObject.java

```diff
@@ -84,6 +84,7 @@
     /**
      * The stream view this object is backed by.
      */
+    @Getter
```

Comment:
Can you please add a note that this getter is for testing purposes or perhaps the annotation VisibleForTesting

Reply:
Sure, I will add a VisibleForTesting function.